src/ui/sandbox-banner.svelte

```diff
@@ -16,17 +18,18 @@
     class="rcb-ui-sandbox-banner"
     {style}
     out:fly={{ y: -100, duration: 300 }}
+    class:isInElement
   >
-    <span class="banner-text">Sandbox</span>
-    <div class="close-button-wrapper">
+    <span class="rcb-sandbox-banner-text">Sandbox</span>
+    <div class="rcb-sandbox-banner-close-button-wrapper">
       <CloseButton on:click={closeBanner} />
     </div>
   </div>
 {/if}
 
 <style>
   .rcb-ui-sandbox-banner {
-    position: sticky;
```

Comment:
minor changes so we don't show the background below when it's closing. 

Sticky does this, and fixed cannot be applied in embedded as it jumps to the viewport